test/serverTest.js

```diff
@@ -9,11 +9,14 @@ const redis = imports.redis
 const server = imports.server
 const XETHRU_STATE = require('../SessionStateXethruEnum.js')
 const MOV_THRESHOLD = 17
+const IM21_DOOR_STATUS = require('../IM21DoorStatusEnum')
+const ST_DOOR_STATUS = require('../SessionStateDoorEnum')
 
 chai.use(chaiHttp)
 
 const testLocation1Id = 'TestLocation1'
 const testLocation1PhoneNumber = '+15005550006'
+const door_coreID = 'door_particlecoreid'
```

Comment:
What do you need this for?

Reply:
this is a parameter in the im21Door mock helper